### DISCUSSION TITLE
Is there a way to change Table View timestamp formatting?

### DISCUSSION THREAD

**Discussion by rwjack** [SigNoz/signoz#2389] [Q&A]
Currently it looks like this:

![image](https://user-images.githubusercontent.com/59068073/221412880-61463157-7987-40f3-8050-72294a2c0179.png)

I would like to have it look a bit simpler, eg:
```
Feb 26th 14:20:02.123
```

**Comment by nityanandagohain** [comment]
As of now, it's not possible. Though I understand your use case.

@pranay01  any thoughts on this.

**Reply by rwjack** [reply]
Perhaps it would be best not to hardcode any format, but to allow the user to chose the format itself, as per:

https://github.com/observiq/ctimefmt/blob/3e07deba22cf7a753f197ef33892023052f26614/ctimefmt.go#L63

Unsure how complicated that would be to implement, but if not, I think we'd definitely need something easier on the eyes.
